### ISSUE TITLE
You have validator issues

### ISSUE THREAD

**Issue by delg0030** [seanokai/portfolio#1]
Please review the following issues noticed by Validator and make those corrections :)
- The web page should have a title that describes topic or purpose (3 results)
- Text elements should have a reasonable contrast ratio (7 results)
